client/components/contact/ContactForm.jsx

```diff
@@ -251,6 +260,7 @@ const ContactForm = () => {
               Submit
             </Button>
           </Grid>
+          <Grid container direction="column" alignItems="center" justifyContent="center" className={classes.footer} />
```

Comment:
Would be great to have a comment explaining why we need to have this.

Reply:
updated